src/pal/tests/palsuite/c_runtime/pow/test1/test1.cpp

```diff
@@ -106,13 +106,19 @@ int __cdecl main(int argc, char **argv)
         { -2.7182818284590452,     1,                     -2.7182818284590452,        PAL_EPSILON * 10 },     // x: -(e)                       expected: e
         { -2.7182818284590452,     PAL_POSINF,             PAL_POSINF,                0 },                    // x: -(e)
 
+        { -1.0,                    PAL_NEGINF,             1.0,                       PAL_EPSILON * 10 },
```

Comment:
No, this is very minor. I just wanted to understand the reason. It's fine to fix them all in one go later.